svm/src/transaction_processor.rs

```diff
@@ -299,31 +299,9 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             .map(|(load_result, tx)| match load_result {
                 Err(e) => TransactionExecutionResult::NotExecuted(e.clone()),
                 Ok(loaded_transaction) => {
-                    let compute_budget = if let Some(compute_budget) = config.compute_budget {
-                        compute_budget
-                    } else {
-                        let mut compute_budget_process_transaction_time =
-                            Measure::start("compute_budget_process_transaction_time");
-                        let maybe_compute_budget = ComputeBudget::try_from_instructions(
-                            tx.message().program_instructions_iter(),
-                        );
-                        compute_budget_process_transaction_time.stop();
-                        saturating_add_assign!(
-                            execute_timings
-                                .execute_accessories
-                                .compute_budget_process_transaction_us,
-                            compute_budget_process_transaction_time.as_us()
```

Comment:
nit: Personally i don't really pay much attention to this metric, cause they are pretty much flat all the time, so removing it is fine with me. But in case there are queries read it, you can move it to `validate_transaction_fee_payer()` to keep "backward compatible". Totally up to you.

Reply:
I'd prefer to remove it, the compute budget processing is now tracked under `"validate_fees_us"` as of https://github.com/anza-xyz/agave/pull/1529